test/key/utils.spec.ts

```diff
@@ -287,7 +201,12 @@ gm9nAQDY//xzc2hy6Efz8NqDJeLg1lh2sZkKcMXP3L+CJbhWJQEAuI6UDakE
 +XVcDsBS+CIi3qg74r/80Ysb7tmRC06znwA=
 =I0d7
 -----END PGP PRIVATE KEY BLOCK-----`});
-    const sessionKey = await generateSessionKeyFromKeyPreferences(key);
-    t.is(sessionKey.data.length, 32);
-    t.is(sessionKey.algorithm, 'aes256');
-});
\ No newline at end of file
+        const sessionKey = await generateSessionKeyFromKeyPreferences(key);
+        expect(sessionKey.data.length).to.equal(32);
+        expect(sessionKey.algorithm).to.equal('aes256');
+    });
+})
+
+
+
+
```

Comment:
This file is hoarding all the newlines :upside_down_face: Some of the other files need some too :cry: